packages/atchops/tools/clean_install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu
+cd ..
+sudo rm -rf build
```

Comment:
This is a bit dangerous, I always try to do this relative to the script in case someone accidentally runs this file from another directory

Reply:
/bin/bash compliant